### ISSUE TITLE
PM4 Monitor - Went Haywire Then Powered Off

### ISSUE THREAD

**Issue by General-IST** [chestertonrc/equipment#6]
As above, PM4 went all weird this evening, during 4 x 1K pieces.

I believe that this has been mentioned before. 

Has any action been taken?

**Comment by southa** [comment]
Have the batteries been changed recently?  If not then that would be the first thing, but after that then replacement is the best bet.  PM5 for £155 inc, which can go on the maintenance budget

**Comment by General-IST** [comment]
NFI!

I am ergo-ing this eve so will have a look on the C2 page for a manual and try to crack it open

**Comment by General-IST** [comment]
![IMG_3205](https://user-images.githubusercontent.com/50678721/58086201-1817fb80-7bb6-11e9-8286-e0fb7441392b.jpg)
![IMG_3207](https://user-images.githubusercontent.com/50678721/58086202-18b09200-7bb6-11e9-9a61-d70b26776d6e.jpg)
![IMG_3204](https://user-images.githubusercontent.com/50678721/58086203-18b09200-7bb6-11e9-8fca-90b87c8deab1.jpg)
![IMG_3206](https://user-images.githubusercontent.com/50678721/58086204-18b09200-7bb6-11e9-8792-dfc0a4adbe8f.jpg)
![IMG_3208](https://user-images.githubusercontent.com/50678721/58086205-19492880-7bb6-11e9-9326-4b5812a6f96c.jpg)

**Comment by General-IST** [comment]
Oops, didn't mean to close the ticket!
Battery doesn't look like it's the issue.
Or does it?
I'm happy to go and buy 2 x Duracells.
Pls advise

**Comment by southa** [comment]
No at 75% the batteries should be fine.  That wire looks dodgy though, I've known one to go nuts when its sensor was shorted out.

I'll order a new one anyway but there might be one in the lockers.  They are not too bad to fit, Concept II web site has the details.

**Comment by General-IST** [comment]
Thanks, I'm happy to fit it.

Cheaper than a whole new PM.

**Comment by southa** [comment]
Sensor ordered.

**Comment by djmcgreal** [comment]
Unit still dodgy. Has a new one arrived?
Ta. Dan.

**Comment by southa** [comment]
Yep the sensor will be in the locker if it hasn't been fitted already.
